GraphConsoleAppV3/GraphRequests.cs

```diff
@@ -0,0 +1,1534 @@
+                Console.WriteLine("\nNew Domain {0} was created", newDomain.Name);
+            }
+            catch (Exception e)
+            {
+                Program.WriteError("\nError creating new Domain {0} : {1}", e.Message,
+                    e.InnerException != null ? e.InnerException.Message : null);
+            }
+            return newDomain;
+
+            #endregion
+        }
+        #endregion
+
+        #region CleanUp
+
+        private static async Task DeleteUser(IUser newUser)
+        {
+            #region Delete user
+
+            //*********************************************************************************************
+            // Delete the user that we just created earlier
+            //*********************************************************************************************
+            if (newUser.ObjectId != null)
+            {
+                try
+                {
+                    await newUser.DeleteAsync();
+                    Console.WriteLine("\nUser {0} was deleted", newUser.DisplayName);
+                }
+                catch (Exception e)
+                {
+                    Program.WriteError("\nError Deleting User {0} {1}", e.Message,
+                        e.InnerException != null ? e.InnerException.Message : "");
+                }
+            }
+
+            #endregion
+        }
+
+        private static async Task DeleteGroup(Group newGroup)
+        {
+            #region Delete Group
+
+            //*********************************************************************************************
+            // Delete the Group that we just created
+            //*********************************************************************************************
+            if (newGroup.ObjectId != null)
+            {
+                try
+                {
+                    await newGroup.DeleteAsync();
+                    Console.WriteLine("\nGroup {0} was deleted", newGroup.DisplayName);
+                }
+                catch (Exception e)
+                {
+                    Program.WriteError("\nError Deleting Group {0} {1}", e.Message,
+                        e.InnerException != null ? e.InnerException.Message : "");
+                }
+            }
+
+            #endregion
+        }
+
+        private static async Task DeleteApplication(Application newApp)
+        {
+            #region Delete Application
+
+            //*********************************************************************************************
+            // Delete Application Objects
+            //*********************************************************************************************
+            if (newApp.ObjectId != null)
+            {
+                try
+                {
+                    await newApp.DeleteAsync();
+                    Console.WriteLine("\nDeleted Application object: " + newApp.ObjectId);
+   
```

Comment:
Added

---
In reply to: [80757697](https://github.com/Azure-Samples/active-directory-dotnet-graphapi-console/pull/37#discussion_r80757697) [](ancestors = 80757697)